packages/shorebird_cli/lib/src/cache.dart

```diff
@@ -176,6 +187,33 @@ allowed to access $storageUrl.''',
   }
 }
 
+class ShorebirdLinkerArtifact extends CachedArtifact {
+  ShorebirdLinkerArtifact({required super.cache, required super.platform});
+
+  @override
+  String get name => 'shorebird_linker';
+
+  @override
+  List<String> get executables => ['shorebird_linker'];
+
+  @override
+  bool get required => false; // Only supported for newer engine revisions.
```

Comment:
I can imagine future us asking: "newer than what?"

Reply:
Yeah good point I can reword to say this is only available in revisions that support mixed-mode